tests/tests/test_artifact.py

```diff
@@ -113,8 +114,8 @@ def test_artifacts_valid(self):
             update = res.updates[0]
             assert len(update.files) == 1
             uf = update.files[0]
-            assert uf.size == len(data)
-            assert uf.checksum
+            assert uf["size"] == len(data)
+            assert uf["checksum"]
```

Comment:
The fact that I had to modify these in the existing tests make me suspect that the changes I made in the API spec are wrong. Please look at it carefully.

Reply:
@lluiscampos The keyword `oneOf` is not in the Swagger 2.0 specification. I don't think Swagger is rich enough to express the desired API.

Reply:
As discussed privately, I will revert the API spec change and workaround it in the tests.

Reply:
Done.